scripts/bootstrap.sh

```diff
@@ -6,8 +6,8 @@ set -o nounset
 
 set -x
 
-SCRIPTS_DIR="$(dirname "${BASH_SOURCE[0]}")"
-ROOT_DIR="$(cd "${SCRIPTS_DIR}/.." && pwd)"
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
```

Comment:
I'm not sure why one would want to do an extra invocation here. Like I guess potentially the scripts dir could be a symlink path and root dir wouldn't be, but you shouldn't be doing file modification on scripts anyway.